packages/flutter/test/widgets/selectable_region_test.dart

```diff
@@ -1707,6 +1708,105 @@ void main() {
     variant: const TargetPlatformVariant(<TargetPlatform>{ TargetPlatform.iOS, TargetPlatform.android }),
   );
 
+  testWidgets('the selection behavior when clicking `Copy` item', (WidgetTester tester) async {
+    List<ContextMenuButtonItem> buttonItems = <ContextMenuButtonItem>[];
+    await tester.pumpWidget(
+      MaterialApp(
+        home: SelectableRegion(
+          focusNode: FocusNode(),
+          selectionControls: materialTextSelectionHandleControls,
+          contextMenuBuilder: (
+            BuildContext context,
+            SelectableRegionState selectableRegionState,
+          ) {
+            buttonItems = selectableRegionState.contextMenuButtonItems;
+            return const SizedBox.shrink();
+          },
+          child: const Text('How are you?'),
+        ),
+      ),
+    );
+
+    final RenderParagraph paragraph1 = tester.renderObject<RenderParagraph>(find.descendant(of: find.text('How are you?'), matching: find.byType(RichText)));
+    final TestGesture gesture = await tester.startGesture(textOffsetToPosition(paragraph1, 6)); // at the 'r'
+    addTearDown(gesture.removePointer);
+    await tester.pump(const Duration(milliseconds: 500));
```

Comment:
Is it possible to use a `pumpAndSettle` here instead of an explicit duration?

Reply:
thank you for you comment, in my opinion, it need to wait for a new frame (gesture) after 500ms. just now I try to call `pumpAndSettle`, but it does't work.

Reply:
Oh I see, this is basically a long press, right?

Could you use [longPressAt](https://master-api.flutter.dev/flutter/flutter_test/WidgetController/longPressAt.html) instead of startGesture?

And you can probably use [kLongPressTimeout](https://master-api.flutter.dev/flutter/gestures/kLongPressTimeout-constant.html) instead of 500ms.

Reply:
nice suggestion~